pyproject.toml

```diff
@@ -37,16 +38,16 @@ license = "Apache 2.0"
 "Youtube" = "https://www.youtube.com/meltano"
 
 [tool.poetry.dependencies]
-python = "<3.11,>=3.7.1"
+python = "<3.12,>=3.7.1"
 backoff = ">=2.0.0,<3.0"
 pendulum = "^2.1.0"
 click = "~=8.0"
 fs = "^2.4.16"
 PyJWT = "~=2.4"
 requests = "^2.25.1"
 cryptography = ">=3.4.6,<39.0.0"
-importlib-metadata = {version = "*", markers = "python_version < \"3.8\""}
-importlib-resources = {version = "^5.9.0", markers = "python_version < \"3.9\""}
+importlib-metadata = {version = "<5.0.0", markers = "python_version < \"3.8\""}
+importlib-resources = {version = "5.9.0", markers = "python_version < \"3.9\""}
```

Comment:
These updates have certainly been problematic. If anyone's curious:

- https://github.com/meltano/sdk/pull/1023#issuecomment-1267796121
- https://github.com/meltano/sdk/pull/1057#issuecomment-1273610294

There's a whole story of maintainer disagreements between `flake8` and `importib-metadata` 😬